crowbar_framework/db/migrate/20150312081459_create_proposals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :barclamp, :null => false
+      t.string :name, :null => false
+      t.text :properties
```

Comment:
> * the revisions and crowbar-applied bits that are in the deployment part should likely just be straight columns in the database anyway

I agree, if we want to keep the revisions, they should be in a column.

> That being said, I'd definitely agree with doing that as a later step; so a research card is fine with me.

Ok, thanks for the explanation. I created a task in the EPIC card for the
research.